package.json

```diff
@@ -21,14 +21,14 @@
     "@serverless-stack/resources": "^1.18.0",
     "@tsconfig/node16": "^1.0.3",
     "@types/git-rev-sync": "^2.0.0",
-    "@ucanto/client": "^4.2.3",
-    "@ucanto/core": "^4.2.3",
-    "@ucanto/principal": "^4.2.3",
-    "@ucanto/transport": "^4.2.3",
+    "@ucanto/client": "^5.1.0",
+    "@ucanto/core": "^5.1.0",
+    "@ucanto/principal": "^5.0.0",
+    "@ucanto/interface": "^6.0.0",
+    "@ucanto/transport": "^5.1.0",
     "@web-std/blob": "^3.0.4",
     "@web-std/fetch": "^4.1.0",
-    "@web3-storage/access": "^9.2.0",
-    "@web3-storage/w3up-client": "^4.1.0",
```

Comment:
This is needed for integration tests

Reply:
But they aren't imported anywhere, how are they used ?

Reply:
https://github.com/web3-storage/w3infra/blob/cd1db3d627c9f294d59bd22f459398115fc98b8a/test/helpers/up-client.js#L9